conf/app/HumanStateVisualizer_iCub3.ini

```diff
@@ -0,0 +1,14 @@
+name                    HumanStateVisualizer
+
+# Model Configuration options
+modelURDFName      "teleoperation_iCub_model_V_3.urdf"
+ignoreMissingLinks  true
+
+# Camera options
+cameraDeltaPosition  (0.0, 2.0, 0.5)
+useFixedCamera       false           # if set to false, the camera follows the model base link
+fixedCameraTarget    (0.0, 0.0, 0.0) # this option is unused when useFixedCamera is false
+maxVisualizationFPS  65
+
+# Remapper Configuration
+humanStateDataPortName "/iCub/RobotStateWrapper/state:o"
```

Comment:
I would suggest adding an empty line at the end.

Reply:
you're right, done